### PR TITLE
Add Unit Tests for EventPublisherMediatorSerializer class

### PR DESCRIPTION
**Pull request by Nirothipan**

### CODE REVIEW

File: modules/core/src/test/java/org/apache/synapse/config/xml/eventing/EventPublisherMediatorSerializerTest.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+     */
+    @Test(expected = SynapseException.class)
+    public void testCreateEventSource1() {
+        EventPublisherMediatorSerializer mediatorSerializer = new EventPublisherMediatorSerializer();
+        EventPublisherMediator mediator = new EventPublisherMediator();
+        mediatorSerializer.serializeSpecificMediator(mediator);
+    }
+
+    @Test
+    public void testCreateEventSource2() {
```
Comment (riyafa) [inline modules/core/src/test/java/org/apache/synapse/config/xml/eventing/EventPublisherMediatorSerializerTest.java]: Shall we add javadoc comment?

Comment (Nirothipan) [reply modules/core/src/test/java/org/apache/synapse/config/xml/eventing/EventPublisherMediatorSerializerTest.java]: fixed

**Comment by Nirothipan** [comment]
Fixed commit message

File: modules/core/src/test/java/org/apache/synapse/config/xml/eventing/EventPublisherMediatorSerializerTest.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+ * Test class for EventPublisherMediatorSerializer
+ */
+public class EventPublisherMediatorSerializerTest {
+
+    /**
+     * Passing unsupported mediator.
+     */
+    @Test(expected = SynapseException.class)
+    public void testCreateEventSource() {
+            EventPublisherMediatorSerializer mediatorSerializer = new EventPublisherMediatorSerializer();
```
Comment (Asitha) [inline modules/core/src/test/java/org/apache/synapse/config/xml/eventing/EventPublisherMediatorSerializerTest.java]: Shall we update the code alignment?

Comment (Nirothipan) [reply modules/core/src/test/java/org/apache/synapse/config/xml/eventing/EventPublisherMediatorSerializerTest.java]: Fixed in https://github.com/wso2/wso2-synapse/pull/1050/files

Comment (Nirothipan) [reply modules/core/src/test/java/org/apache/synapse/config/xml/eventing/EventPublisherMediatorSerializerTest.java]: Foxed in https://github.com/wso2/wso2-synapse/pull/1051

**Pull request merged**